pkgdb2/lib/__init__.py

```diff
@@ -1972,15 +1975,21 @@ def add_new_package_request(
     if pkg_collection.startswith(('el', 'epel')):
         _validate_pkg(session, pkg_collection[-1:], pkg_name)
 
+    if pkg_description:
```

Comment:
That's a bug then, it should be: `` pkg_description.strip() if pkg_description and pkg_description.strip() else None``